### PR TITLE
Make no change. @bontgoy

### PR DESCRIPTION
**Pull request by AndreeaBarba**

### CODE REVIEW

File: ruby/vampires.rb [hunk 1]
```diff
@@ -56,5 +56,4 @@
   end
```
Comment (Bontgoy) [inline ruby/vampires.rb:56]: Line 15:

I would like to see you follow the age convention in the instructions.  We want to know if you the employee is lying about their age.

Lets consider checking the current year against how old they say they are and what year they say that they were born in.  Investigate the Ruby `Time` class and see if you can come up with something creative and dynamic.

Comment (Bontgoy) [inline ruby/vampires.rb:56]: Line 47:

For your while loop lets add a condition that will help the loop break naturally.  The functionality is definitely working in this loop but `break` is a little abrupt.  We always want a flow to our loops so providing a variable to evaluate would be a good refactor idea here.

Comment (Bontgoy) [inline ruby/vampires.rb:56]: Inside your line 47 loop:

This section should happen before the final vampire evaluations.  Try to keep your logic organized in a way that the program collects all of the necessary data first and then does a full evaluation.

Collecting the allergy data first and then running returning the vampire results all at once is a better practice vs collecting some of the data, returning an eval and then re-evaluating.

**Review by Bontgoy — COMMENTED** [review]
@barbaselua Thanks for sending this over.  I see what happened last time.

When tagging me make sure to tag me in the comment section and not the header.  Tagging me in the comment section notifies me with a link to the PR.  Tagging me in the header just tells me that you tagged me somewhere in github but gives me no info as to where.

File: ruby/vampires.rb (hunk 1 above)
Comment (AndreeaBarba) [reply ruby/vampires.rb:56]: @Bontgoy Thank you for the review, Tyler! I had my doubts about where in the code I should place the allergy part of the challenge. I guess I should've just asked for a clarification. I will take your advice and refactor my code.

**Pull request merged**